docs/graphql/graphql-spans.md

```diff
@@ -9,10 +9,9 @@ linkTitle: GraphQL Server
 This document defines semantic conventions to apply when instrumenting the GraphQL implementation. They map GraphQL
 operations to attributes on a Span.
 
-The **span name** MUST be of the format `<graphql.operation.type> <graphql.operation.name>` provided that
-`graphql.operation.type` and `graphql.operation.name` are available. If `graphql.operation.name` is not available, the
-span SHOULD be named `<graphql.operation.type>`. When `<graphql.operation.type>` is not available, `GraphQL Operation`
-MAY be used as span name.
+The **span name** SHOULD be of the format `<graphql.operation.type>` provided that
+`graphql.operation.type` is available. If `graphql.operation.type` is not available, the
+span SHOULD be named `GraphQL Operation`.
```

Comment:
@lmolkova @joaopgrassi @robertlaurin - Based on your earlier discussion, how does this update sound?

```suggestion
The **span name** SHOULD be of the format `<graphql.operation.type>` provided
`graphql.operation.type` is available. If `graphql.operation.type` is not available, the
span SHOULD be named `GraphQL Operation`.

In addition, Instrumentation MAY provide a configuration option to add 
`<graphql.operation.name>` to the span name, making the name 
`<graphql.operation.type> <graphql.operation.name>`. In this case, if 
`<graphql.operation.name>` is not available, the span SHOULD be named 
`<graphql.operation.type>`.
```

Reply:
looks good!

A small readability suggestion, feel free to ignore if you don't like it.

> [!NOTE] 
> The `graphql.operation.name` value is provided by the client and can have high cardinality. Using it in GraphQL server span name (by default) is NOT RECOMMENDED.
>
> Instrumentation MAY provide a configuration option to enable more descriptive span name following `{graphql.operation.type} {graphql.operation.name}` format when `graphql.operation.name` is available.

Reply:
BTW, we use `{}` for templates instead of `<>`  in other documents.

Reply:
I like those suggestions! How's this? 

```suggestion
The **span name** SHOULD be of the format `{graphql.operation.type}` provided
`graphql.operation.type` is available. If `graphql.operation.type` is not available, the
span SHOULD be named `GraphQL Operation`.

> **Warning**
> The `graphql.operation.name` value is provided by the client and can have high cardinality. Using it in the GraphQL server span name (by default) is NOT RECOMMENDED.
>
> Instrumentation MAY provide a configuration option to enable a more descriptive span name following `{graphql.operation.type} {graphql.operation.name}` format when `graphql.operation.name` is available.
```

Reply:
thanks! looks good!